### ISSUE TITLE
Update `HACKING.md` with `master` vs `experimental`

### ISSUE THREAD

**Issue by JacquesCarette** [agda/agda-stdlib#313]
Doing `make test` fails in `Reflection.agda` with

```
  Checking Reflection (/Users/carette/agda-stdlib-fork/src/Reflection.agda).
/Users/carette/agda-stdlib-fork/src/Reflection.agda:138,1-143,21
Incomplete pattern matching for showLiteral. Missing cases:
  showLiteral (Literal.word64 n)
when checking the definition of showLiteral
/Users/carette/agda-stdlib-fork/src/Reflection.agda:325,1-360,21
Incomplete pattern matching for _≟-Lit_. Missing cases:
  Literal.word64 n ≟-Lit nat n₁
  x ≟-Lit Literal.word64 n
  Literal.word64 n ≟-Lit float x
  Literal.word64 n ≟-Lit char c
  Literal.word64 n ≟-Lit string s
  Literal.word64 n ≟-Lit name x
  Literal.word64 n ≟-Lit meta x
when checking the definition of _≟-Lit_
```

When trying to do some work on stdlib, which version of Agda should I be using?  `HACKING.md` is silent on the matter.

**Comment by MatthewDaggitt** [comment]
So if you're using the development version of Agda you need to use the `experimental` branch. If you're using the current version of Agda (i.e. 2.5.3) you want to use the `master` branch.

However all changes should be made on master if they don't **require** the latest development version of Agda.

You're right this should be `HACKING.md`.

**Comment by JacquesCarette** [comment]
Ok, in my case, I wanted 2.5.3+experimental.  Time to re-re-install.

**Comment by MatthewDaggitt** [comment]
Closed with f394c57
